home/components/SmallProjectCard.js

```diff
@@ -50,9 +51,15 @@ export default class SmallProjectCard extends React.PureComponent {
         <View style={[styles.infoContainer, !this.props.fullWidthBorder && styles.bottomBorder]}>
           <View style={styles.projectNameContainer}>
             <View style={{ flex: 1, flexGrow: 4 }}>
-              <Text style={styles.projectNameText} ellipsizeMode="tail" numberOfLines={1}>
-                {projectName}
-              </Text>
+              { platform != null ?
```

Comment:
In this case we know `platform` will always be "native" or "web" and we aren't specifically testing for null or undefined so a falsy check is fine:

```suggestion
              { platform ?
```

Reply:
I put this in for the case where the user hadn't updated their cli client so they wouldn't see an empty '[]'. Perhaps no need to cater to the transition?